src/modals/confirmation-modal/operations-confirmation/operations-confirmation.tsx

```diff
@@ -34,6 +34,7 @@ interface Props {
 export const OperationsConfirmation: FC<Props> = ({ sender, opParams, onSubmit, onBackButtonPress, children }) => {
   const styles = useOperationsConfirmationStyles();
   const dispatch = useDispatch();
+  const [estimationsRequested, setEstimationsRequested] = useState(false);
```

Comment:
let's use `is` prefix for boolean falgs 
`[isEstimationsRequested, setIsEstimationsRequested]`